lti_consumer/forms.py

```diff
@@ -0,0 +1,46 @@
+"""
+Defines a form for providing validation of LTI consumer course-specific configuration.
+"""
+import logging
+
+from django import forms
+from opaque_keys import InvalidKeyError
+from opaque_keys.edx.locator import CourseLocator
+
+# TODO: find a way to access the modulestore..
+from xmodule.modulestore.django import modulestore
```

Comment:
For the `modulestore`, as I mentioned above, @bmedx is actively working on pulling it out of edx-platform.  Once he does so, you'll be able to use it here.

In the meantime, perhaps you can disable the validation of the course-key in this code.